homeassistant/components/group/sensor.py

```diff
@@ -255,6 +258,19 @@ def calc_last(
     return attributes, last
 
 
+def calc_first(
+    sensor_values: list[tuple[str, float, State]],
+) -> tuple[dict[str, str | None], float | None]:
+    """Calculate first value."""
+    first_entity_id: str | None = None
+    first: float | None = None
+    if len(sensor_values) > 0:
```

Comment:
Empty sequences are false. Should be condensed to:
```suggestion
    if sensor_values:
```